content/en/data-playground/noaa.md

```diff
@@ -24,7 +24,228 @@ weight: 2
 
 {{% data-playground-code-tabs %}}
 
+{{% data-playground-code-tab tabIndex="python" name="Python"  %}}
+
+```python
+import json
+import asyncio
+import warnings
+from datetime import datetime
+
+import requests
+from pyensign.events import Event
+from pyensign.ensign import Ensign
+
+# TODO Python>3.10 needs to ignore DeprecationWarning: There is no current event loop
+warnings.filterwarnings("ignore")
+
+# TODO: replace with YOU - your email and app details :)
+ME = "(https://rotational.io/data-playground/noaa/, weather@rotational.io)"
+
+# TODO: these are memorable for ME. Replace with the locations of interest to YOU
+LOCS = {
+    "north_pole": {
+        "lat": "64.7511",
+        "long": "-147.3494"
+    },
+    "cafe_du_monde": {
+        "lat": "29.957684",
+        "long": "-90.061892"
+    },
+}
 
+class WeatherPublisher:
```

Comment:
Will this code also be added to data-playground repo as a `NOAA` example? There was a brief discussion about displaying the data that will be returned in the code snippets, so I'm wondering if we still need to decide how the code snippets will differ from what will be added to the repos.

Reply:
I snuck it in [here](https://github.com/rotationalio/data-playground/tree/main/weather) 😉 

But it's a great point, since I'm also not totally sure what we should be showing on the Data Playground webpage vs Github. For this PR I decided to just use the Publisher from the Github repo in the  Data Playground webpage example. 

But I guess I'm realizing that the looooong code snippets don't really make sense in the webpage. Something to consider as we continue to evolve the Playground?